content/_jobs/2019-09-27-dinsic-portfolio.md

```diff
@@ -0,0 +1,54 @@
+---
+roles: plusieurs responsables de portefeuilles
+title: "L'incubateur de la DINSIC recherche : plusieurs responsables de portefeuilles"
+open: true
+---
+
+L'incubateur de la DINSIC recrute des responsables de portefeuilles pour participer à la mise en place d'une équipe dédiée, autonome et pluridisciplinaire qui se donne pour mission de participer à la réussite du programme beta.gouv.fr
+
+<!--more-->
+
+## Le contexte
+
+La communauté beta.gouv.fr à, depuis la création de la première Startup d'État, décidé collectivement que la meilleure manière de mettre en oeuvre une politique publique passe par la constitution d'une équipe autonome. Cette équipe est elle-même constituée de membres issus du public ou du privé qui se donnent pour mission de résoudre les problèmes de ses usagers. Elle pilote son action au jour le jour pour atteindre son objectif. Et enfin, elle répond des ses actes de façon transparente envers ses sponsors et ses usagers.
+
+Nous souhaitons construire, au sein de l'incubateur de la DINSIC, une équipe dédiée à l'accompagnement de ce programme qui respectera ce manifeste. Cette équipe aura pour mission de garantir à toutes les équipes de la communauté, celles de Startup d’État ainsi que celles des incubateurs ministériels qui le souhaitent, les conditions nécessaires à leur réussite, à diffuser cette culture dans de nouvelles administrations et à favoriser la naissance de produits innovants.
+
+## Missions
+
+- Tout faire pour garantir le succès des produits : se concentrer sur l'impact et les besoins des usagers, bonnes métriques, itération rapide, solutions aux goulots d'étranglement, anticiper le déploiement… ;
```

Comment:
J'aurais une préférence pour « itération courte » plutôt que « rapide », et j'enlèverais « solutions aux goulots d'étrangement », peut-être pour mettre « solutions aux problèmes » (ou rien)

Reply:
> se concentrer sur l'impact

=> **concentration** sur l'impact

(pour être cohérent avec l'absence de verbe dans les autres éléments de cette phrase)

> bonnes métriques

=> **des** métriques **qui les reflètent** ?

C'est un point mineur, c'est peut-être mieux de laisser une formule plus concise.

Reply:
De ce que je comprends du rôle, trouver des _« solutions aux goulots d'étrangement »_ c'est une partie très importante (en anglais en parle de _dealing with red tape_). 

Peut-être préciser s'il s'agit de :

1. Goulots d'étrangement internes à l'équipe (staffing, communication...) ; et/ou
2. Externes : conventions, avenants, bons de commande, problèmes avec les sponsors...